forms/src/main/java/org/devgateway/toolkit/forms/wicket/page/BasePage.java

```diff
@@ -141,7 +141,7 @@ protected Navbar newNavbar(final String markupId) {
 		NavbarButton<LogoutPage> logoutMenu = new NavbarButton<LogoutPage>(LogoutPage.class,
 				new StringResourceModel("navbar.logout", this, null));
 		logoutMenu.setIconType(GlyphIconType.logout);
-		MetaDataRoleAuthorizationStrategy.authorize(logoutMenu, Component.RENDER, SecurityConstants.Roles.ROLE_EDITOR);
```

Comment:
1) you need to rename the roles in the liquibase-changelog.xml which are used to pupulate the role database.

2) you need to take in account here that projects using dg-toolkit already the ROLE_EDITOR inserted into database into table ROLE. It would be advisable to rename the roles in database using liquibase, otherwise existing users will have another version of the roles in the database